dcos-aws-cli/1.9/README.md

```diff
@@ -21,12 +21,22 @@ $ dcos package install dcos-aws-cli
 
 ## Usage
 
-The DC/OS AWS CLI currently supports only one command:
+The DC/OS AWS CLI currently supports only two commands:
 - Listing of external IP addresses
-Retrieving the externally resolvable IP addresses of public Agents can be difficult, as DC/OS internally used the interal IP addresses. This command will list the external IP addresses for all public agents.
+Retrieving the externally resolvable IP addresses of public Agents can be difficult, as DC/OS internally used the internal IP addresses. This command will list the external IP addresses for all public agents.
 
 ```bash
 $ dcos dcos-aws-cli publicIPs
+35.158.189.26
+```
+
+- List all applications and their ports running on public agents
+Checking all apps running on public agents and their ports might is still a tedious task. This command lists all apps which have reserved port resources on a public Agent. Note that this does not--yet--include applications which are exposed via marathon-lb or applications which use unreserved ports.
```

Comment:
Not sure what 'might' is doing there - think this should be 'can be a tedious task' ?